src/parse.ts

```diff
@@ -71,7 +73,7 @@ export class PythonParser {
         let line_num: number = position.line;
         let def_indentation: number = this.getIndentation(document.lineAt(line_num - 1));
 
-        while (line_num < document.lineCount - 1) {
+        while (line_num < document.lineCount) {
```

Comment:
Solves the return not detected at the last line issue mentioned in [Issue #3](https://github.com/NilsJPWerner/autoDocstring/issues/3)